README.md

```diff
@@ -41,9 +39,10 @@ vi ~/.tailpipe/config/github.tpc
 ```
 
 ```hcl
-partition "github_audit_log" "my_logs" {
+partition "github_audit_log" "audit_log" {
   source "file"  {
-    paths = ["/Users/myuser/github_logs"]
+		paths = ["/Users/dir/path"]
+		file_layout = "export-turbot-%{NUMBER:prefix}.json"
```

Comment:
```suggestion
		file_layout = "%{DATA}.json.gz"
```

We can make this generic so users can copy and paste more easily (they will still need to change the dir though)